src/pci_lmt/results.py

```diff
@@ -84,3 +102,33 @@ def write(self, result: LmtLaneResult) -> None:
             self.__emitted_header = True
 
         print(row)
+
+
+class OctTvOutputReporter(Reporter):
+    def __init__(self):
+        # TODO(sksekar): Add support for saving to file.
+        self._run: tv.TestRun = None
+        self._step: tv.TestStep = None
+
+    def start_run(self, host: HostInfo, version: str) -> None:
```

Comment:
this api design allows you to call `end_run` before `start_run`, in which case, code panics since `self._run` will be None

Reply:
Good catch. There are ways to avoid (check for None) but that would make the code clumsy. Would like to wait for your feedback/guidance in other comments before fixing this.

Reply:
Addressed this by using contextmanagers